rules/early-return/tests/Rector/If_/ChangeAndIfToEarlyReturnRector/Fixture/always_return.php.inc

```diff
@@ -0,0 +1,38 @@
+<?php
+
+namespace Rector\EarlyReturn\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+class AlwaysReturn
+{
+    public function run(&$queries, $sql)
+    {
+        if (!empty($sql) && preg_match('@[^[:space:]]+@', $sql)) {
+            $queries[] = ['query' => $sql, 'empty' => 'empty'];
+        }
+
+        return true;
+    }
+}
+
+?>
+-----
+<?php
+
+namespace Rector\EarlyReturn\Tests\Rector\If_\ChangeAndIfToEarlyReturnRector\Fixture;
+
+class AlwaysReturn
+{
+    public function run(&$queries, $sql)
+    {
+        if (empty($sql)) {
```

Comment:
not sure whether the resulting code is more readable in contradt to what the IS code was, but functionality wise this LGTM

Reply:
We're open to configurable parameter to improve readability :+1: